eng/common/TestResources/README.md

```diff
@@ -89,6 +89,12 @@ Remove-AzADServicePrincipal -ApplicationId $sp.ApplicationId -Force
 
 If you persisted environment variables, you should also remove those as well.
 
+Some scripts accept the `AdditionalParameters` parameter to control additional resource configuration options. For example:
+
+```powershell
+.\eng\common\TestResources\New-TestResources.ps1 keyvault -AdditionalParameters @{enableHsm = $true}
```

Comment:
Like other samples, don't include the path. This assumes too much of the CWD of users. It's not common in PowerShell documentation in general.